firestore/firestore_snippets/query.go

```diff
@@ -249,6 +249,16 @@ func createMultipleStartAtQuery(client *firestore.Client) {
 	// [END fs_start_at_multiple]
 }
 
+func createInQuery(ctx context.Context, client *firestore.Client) error {
+	cities := client.Collection("cities")
+	// [START fs_in_query]
+	query := cities.Where("regions", "in", [][]string{{"west_coast"}, {"east_coast"}}).Documents(ctx)
+	// [END fs_in_query]
+
+	_ = query
+	return nil
+}
+
```

Comment:
Can you also add a function for In query with an array of arrays? For example in java: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/firestore/src/main/java/com/example/firestore/snippets/QueryDataSnippets.java#L464-L475 This scenario is specifically called out in the docs.

Reply:
This is an array of arrays (I directly imitated the example in the docs here: https://firebase.google.com/docs/firestore/query-data/queries#in_and_array-contains-any )

Can also add one mirroring https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/firestore/src/main/java/com/example/firestore/snippets/QueryDataSnippets.java#L455 as well if you like.

Reply:
Yeah, we want both scenarios. Sorry for the confusion.